.markdownlint.json

```diff
@@ -0,0 +1,12 @@
+{
+  "default": true,
+  "MD004": false,
+  "MD013": false,
+  "MD025": false,
+  "MD026": false,
+  "MD029": false,
```

Comment:
This is another good rule to keep. There's only one place where this was violated, and it was due to a missing newline between the list sections, not an actual violation.